api/internal/accumulator/resaccumulator_test.go

```diff
@@ -224,20 +225,26 @@ func TestResolveVarConflicts(t *testing.T) {
 	// create accumulators holding apparently conflicting vars that are not
 	// actually in conflict because they point to the same concrete value.
 	rm0 := resmap.New()
-	rm0.Append(rf.FromMap(fooAws))
+	err := rm0.Append(rf.FromMap(fooAws))
+	require.NoError(t, err)
```

Comment:
`require.NoError(t, rm0.Append(rf.FromMap(fooAws)))`, ditto throughout this file and wherever possible